.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+- 3.6
+- 3.5
+- 2.7
+install: pip install -U tox-travis codecov
+script: tox
+
+deploy:
+  provider: pypi
+  distributions: sdist bdist_wheel
+  user: lincrosenbach
```

Comment:
Please add me as `maintainer` on PyPI as well. 👍

Reply:
@igortg Please add us as Mantainer as you are the Owner of the project.

Reply:
done